src/service/cli/generate.js

```diff
@@ -67,30 +27,40 @@ const PictureRestrict = {
 };
 
 const getPictureFileName = (randomInt) => {
-  return `item${(`0${randomInt}`).substr(-(PictureRestrict.toString()).length)}.jpg`;
+  return `item${(`0${randomInt}`).substr(-(PictureRestrict.max.toString()).length)}.jpg`;
```

Comment:
Диковато выглядит, эта запись эквивалентна
```suggestion
    return `item${number.padStart(2, "0")}.jpg`;
```

Reply:
Не встречал такой метод, заменю.